packages/pages/src/util/runtime.ts

```diff
@@ -9,22 +11,26 @@ class Runtime {
   version: string;
 
   constructor() {
-    if (typeof process !== "undefined") {
-      this.name = "node";
-      this.version = process.versions.node;
-      this.isServerSide = true;
-    } else if (typeof window !== "undefined" && !("Deno" in window)) {
-      this.name = "browser";
-      this.version = navigator.userAgent;
-      this.isServerSide = false;
-    } else {
+    if (isDeno) {
       this.name = "deno";
       this.version = "";
-      if (typeof window !== "undefined") {
+      if (isBrowser) {
         this.version = (window as any).Deno?.version.deno || "";
       }
       this.isServerSide = true;
+      return;
+    }
+
+    if (isNode) {
```

Comment:
Nit but can we use an `else if` or switch statement? I miss the return statement sometimes when I read this.